package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>dbow2</name>
```

Comment:
If there is no restriction about capitalization, could you change the name to `DBoW2`?

Reply:
The ROS community tries to follow common C variable naming conventions when assigning rosdep keys, as we package software using the same package name for consistency:  

* http://wiki.ros.org/ROS/Patterns/Conventions#Packages
* https://www.debian.org/doc/manuals/debmake-doc/ch05.en.html#name-version

I'd recommend keeping with lowercase to avoid future issues.